corehq/apps/app_manager/models.py

```diff
@@ -2547,6 +2549,10 @@ def detail_sort_elements(self):
         except Exception:
             return []
 
+    @property
+    def search_detail(self):
```

Comment:
What would you think of asserting `module_offers_search` here to be extra defensive?

Reply:
I don't think anything bad would happen if this was called even if there was no search action. This property also doesn't have any side effects, so I think it should be left up to the caller to determine whether they want to use the value or not.